packages/imodel-react-hooks/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@bentley/imodel-react-hooks",
+  "version": "0.1.0",
+  "description": "Reusable React Hooks to help with iModel.js",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "keywords": [
+    "Bentley",
+    "hooks",
+    "react",
+    "iModel.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/imodel-react-hooks"
+  },
+  "license": "MIT",
+  "private": false,
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib/**/*",
+    "build/**/*"
```

Comment:
remove

Reply:
build is all in the clean command and in the tsconfig

Reply:
I assume you mean remove the "files" subobject?